packages/artifact/src/upload-artifact-http-client.ts

```diff
@@ -0,0 +1,307 @@
+import {debug} from '@actions/core'
+import {HttpClientResponse, HttpClient} from '@actions/http-client/index'
+import {IHttpClientResponse} from '@actions/http-client/interfaces'
+import {
+  CreateArtifactResponse,
+  CreateArtifactParameters,
+  PatchArtifactSize,
+  PatchArtifactSizeSuccessResponse,
+  UploadResults
+} from './contracts'
+import * as fs from 'fs'
+import {SearchResult} from './search'
+import {UploadOptions} from './upload-options'
+import {URL} from 'url'
+import {
+  createHttpClient,
+  getArtifactUrl,
+  getContentRange,
+  getRequestOptions,
+  isRetryableStatusCode,
+  isSuccessStatusCode,
+  parseEnvNumber
+} from './utils'
+import {getRuntimeToken, getRuntimeUrl, getWorkFlowRunId} from './env-variables'
+
+const defaultChunkUploadConcurrency = 3
+const defaultFileUploadConcurrency = 2
+
+/**
+ * Step 1 of 3 when uploading an artifact. Creates a file container for the new artifact in the remote blob storage/file service
+ * @param {string} artifactName Name of the artifact being created
+ * @returns The response from the Artifact Service if the file container was succesfully created
+ */
+export async function createArtifactInFileContainer(
+  artifactName: string
+): Promise<CreateArtifactResponse> {
+  const parameters: CreateArtifactParameters = {
+    Type: 'actions_storage',
+    Name: artifactName
+  }
+  const data: string = JSON.stringify(parameters, null, 2)
+  const artifactUrl = getArtifactUrl(getRuntimeUrl(), getWorkFlowRunId())
+  const client = createHttpClient(getRuntimeToken())
+  const requestOptions = getRequestOptions(
+    'application/json',
```

Comment:
Initially one was for the `Accept` header, the other was for the `Content-Type` header. I've removed the `Accept` header parameter and just set it to `application/json` since all of our calls use it. Cleans up the code a little bit